src/Tizen.WebView/Tizen.WebView/Settings.cs

```diff
@@ -98,5 +115,75 @@ public int DefaultFontSize
                 Interop.ChromiumEwk.ewk_settings_default_font_size_set(_handle, value);
             }
         }
+ 
+        /// <summary>
+        /// Whether the script can open windows.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public bool ScriptsCanOpenWindows
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_settings_scripts_can_open_windows_get(_handle);
+            }
+
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_scripts_can_open_windows_set(_handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Whether the view port meta tag enabled.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public bool ViewPortMetaTagEnabled
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_settings_viewport_meta_tag_get(_handle);
+            }
+
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_viewport_meta_tag_set(_handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Whether the autofill password form enabled.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public bool AutoFillPasswordFormEnabled
+        {
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_autofill_password_form_enabled_set(_handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Whether the form candidate data enabled.
+        /// </summary>
+        /// <since_tizen> 4 </since_tizen>
+        public bool FormCandidateDataEnabled
+        {
+            set
+            {
+                Interop.ChromiumEwk.ewk_settings_form_candidate_data_enabled_set(_handle, value);
+            }
+        }
+
+        /// <summary>
+        /// Whether the form profile data enabled.
```

Comment:
What is `FormProfileData` ?

Reply:
it is AutoFill form

Reply:
It is a just example to use enum class to control autofill option

``` c#
	[Flags]
	public enum AutoFillMode
	{
		None = 0,
		Forms = 1 << 1,
		Password = 1 << 2,
		Suggestion = 1 << 3,
	}

    void SetAutoFillMode(AutoFillMode mode) 
    {
    	Interop.ChromiumEwk.ewk_settings_form_profile_data_enabled_set(_handle, mode.HasFlag(AutoFillMode.Forms));
		Interop.ChromiumEwk.ewk_settings_autofill_password_form_enabled_set(_handle, mode.HasFlag(AutoFillMode.Password));
		Interop.ChromiumEwk.ewk_settings_form_candidate_data_enabled_set(_handle, mode.HasFlag(AutoFillMode.Suggestion));
    }

```

Reply:
Done